### PR TITLE
Addition of Custom Test Users for Hypertension Fasttrack

### PR DESCRIPTION
**Pull request by ramosan-lit**

### CODE REVIEW

**Review by eclendenning-va — APPROVED** [review]

**Review by jblefkove-va — CHANGES_REQUESTED** [review]
hol up

File: datamart/dm-records-1008711076V809443/dmMedOrd1008711076V809443_1.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"objectType":"MedicationOrder","objectVersion":"1","cdwId":"1008711076V809443_1:O","patient":{"type":"Patient","reference":"1008711076V809443","display":"Mr. Jesse Gray"},"dateWritten":"2020-08-14T08:00:00Z","status":"active","prescriber":{"type":"Practitioner","reference":"416704:S","display":"DR. JANE460 DOE922 MD"},"medication":{"type":"Medication","reference":"12816:L","display":"Amlodipine 5 MG Oral Tablet"},"dosageInstruction":[{"dosageText":null,"timingText":"Once Per Day","additionalInstructions":null,"asNeeded":false,"routeText":"ORAL","doseQuantityValue":1.0,"doseQuantityUnit":null},{"dosageText":null,"timingText":"Twice Per Day","additionalInstructions":null,"asNeeded":false,"routeText":"ORAL","doseQuantityValue":1.0,"doseQuantityUnit":null}],"dispenseRequest":{"numberOfRepeatsAllowed":null,"quantity":1.0,"unit":null,"expectedSupplyDuration":30,"supplyDurationUnits":"days"}}
```
Comment (jblefkove-va) [inline datamart/dm-records-1008711076V809443/dmMedOrd1008711076V809443_1.json:1]: `cdwId` should be `number:suffix` (number is arbitrary as long as it does not conflict with existing records)

File: datamart/dm-records-1008711084V002120/dmPat1008711084V002120.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{"objectType":"Patient","objectVersion":1,"fullIcn":"1008711084V002120","ssn":"796-37-8782","name":"Mr. Ralph Lee","lastName":"Lee","firstName":"Ralph","birthDateTime":"1948-10-30T00:00:00Z","deceased":"No","gender":"M","managingOrganization":"561596:I","ethnicity":{"display":"UNKNOWN BY PATIENT","abbrev":"U","hl7":"9999-4"},"race":[{"display":"White"}],"telecom":[{"type":"Patient Cell Phone","phoneNumber":"8008271000"}],"address":[{"street1":"793 Kub Club","city":"Hampton","state":"Virginia","postalCode":"23605"}]}
```
Comment (jblefkove-va) [inline datamart/dm-records-1008711084V002120/dmPat1008711084V002120.json:1]: Do we need the middle name, `Ralph E Lee`?

**Review by jblefkove-va — APPROVED** [review]
:shipit:

**Pull request merged**